### PR TITLE
Fix PVS-Studio issues related to 'Recoveryservices'

### PR DESCRIPTION
**Pull request by Youri970410**
## Description

Fix PVS-Studio issues related to 'Recoveryservices'
Fix #10223

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs [hunk 1]
```diff
@@ -261,9 +261,9 @@ private void StartRPITestFailover()
                 }
                 else
                 {
-                    new ArgumentException(
+                    Console.WriteLine(new ArgumentException(
```
Comment (wyunchi-ms) [inline src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs:264]: Please do not modify these exceptions to `Console.WriteLine` but add a `TODO` item to notify service team to change them.

**Comment by wyunchi-ms** [comment]
Please take a look @vidyadharijami

**Comment by vidyadharijami** [comment]
> Please take a look @vidyadharijami

Hi @wyunchi-ms,

Is there any time line to take up this issue ? Can we take this up in upcoming powershell release.

Adding @hiaga for backup service changes. And I will look into the changes for Site recovery service.

File: src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs (hunk 1 above)
Comment (Youri970410) [reply src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/StartAzureRmRecoveryServicesAsrTestFailoverJob.cs]: Got it and fixed it.Thanks.

**Comment by hiaga** [comment]
@vidyadharijami, I'll take up these changes in upcoming release.

**Comment by wyunchi-ms** [comment]
Hi @hiaga , throw an new exception will cause Breaking Change. So we suggest to add some notification in the cmdlets and do the breaking changes in next breaking change release.
1. You can find some guideline [here](https://github.com/Azure/azure-powershell-pr/wiki/How-to-Release-a-Breaking-Change).
2. I will merge this PR this time. You can find these exceptions by searching `TODO`.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**